packages/relay/src/lib/services/hbarLimitService/index.ts

```diff
@@ -181,6 +191,10 @@ export class HbarLimitService implements IHbarLimitService {
     requestDetails: RequestDetails,
     estimatedTxFee: number = 0,
   ): Promise<boolean> {
+    if (this.disableRateLimiter) {
```

Comment:
I would suggest adding an `isEnabled()` function to `IHbarLimitService`.
Then this method can call it and it can be tested.

Reply:
Done.